### DISCUSSION TITLE
Error in Daily Quote

### DISCUSSION THREAD

**Discussion by Lecocarvalho** [SilentVoid13/Templater#1461] [Help]
Hello everyone, how are you? In the past few weeks, I have been having problems with the Web Module of Templater related to the Daily Quote.
I would like to know if anyone else is facing this issue and/or if there is any expected resolution for this case?
![Captura de Tela 2024-09-23 às 16 35 11](https://github.com/user-attachments/assets/f02a81a8-bd9a-46a6-a9d7-377d64f78578)

**Comment by jkstress** [comment]
Same for me, just yesterday, the normal templater codes stopped working too: Templater Error: Template parsing error, aborting. 
 Cannot access 'moment' before initialization

**Reply by Zachatoo** [reply]
Please make a separate discussion if you want help with this. Your issue and the daily quotes issue are not related, the daily quotes issue is known and has been broken for a while, I haven't heard of the error you're describing. You'll need to provide your template and Templater settings.

**Comment by chance-nelson** [comment]
I have been experiencing the same issue. The root cause appears to be an expired SSL cert with the quotes API Templater uses. I have published a PR that _should_ solve that issue (#1459), pending review, and you can find workarounds to make manual API calls to other quote services in this issue: #1407
